Makefile

```diff
@@ -76,6 +76,8 @@ endif
 BUILDER_SUFFIX=$(shell echo $(PREFIX) | cut -d '/' -f1)
 
 publish:
+	docker login projects.registry.vmware.com/tanzu_observability_keights_saas
```

Comment:
We `docker login` to `tanzu_observability` registry as a separate step in release pipeline, and then `make publish`. I am also concerned this change may break our release pipeline.

Reply:
I am wondering if we could get away with doing `docker login $PREFIX ` instead. It could be that the [ci.Jenkinsfile's login would first execute](https://github.com/wavefrontHQ/wavefront-collector-for-kubernetes/blob/master/ci.Jenkinsfile#L47) and later a non credential login in makefile, might not affect it and just be redundant. While this might be helpful for local running of `make publish`.

Reply:
Looks like that (i.e., `docker login $PREFIX`) would work based on the already [run CI log](https://tobs-k8s-saas-ci.svc.eng.vmware.com/blue/organizations/jenkins/wavefront-collector-for-kubernetes-ci/detail/PR-352/1/pipeline/45/#step-59-log-1)

Reply:
good call on $PREFIX, @priyaselvaganesan.

Reply:
@helen-shao , when I run this locally and am already authenticated, I just get this: "Authenticating with existing credentials...". If we are already registered in our release pipeline, would you expect it to do the same there?

Reply:
I see that the [release pipeline](https://github.com/wavefrontHQ/wavefront-collector-for-kubernetes/blob/master/release.Jenkinsfile#L100) is going to need `$PREFIX` in the `Publish GA Docker Hub` stage's login. I am wondering if there was a conscious reason to not keep [these two lines](https://github.com/wavefrontHQ/wavefront-collector-for-kubernetes/blob/master/release.Jenkinsfile#L100-L101) for publishing to docker hub consistent with publishing to harbor. If it wasn't conscious, we would need them to be consistent for this PR to work  :grin:

Reply:
Live code reviewed with Priya, got the OK. YAY!